src/main.ts

```diff
@@ -136,7 +136,7 @@ class NpmCodeActionProvider implements CodeActionProvider {
 			if (diag.source === 'npm') {
 				let result = /^Module '(\S*)' is not installed/.exec(diag.message);
 				if (result) {
-					const moduleName = result[1];
+					const moduleName = result[1]; 
 					addFixNpmInstallModule(cmds, moduleName);
 					addFixNpmInstall(cmds);

```

Comment:
test comment

Reply:
This is long discussionsf sdf with many lnseres jsfl ksadjfl;ksad fjlsk;adf ;sfkjasdl k;fjasdl;kf jsadlkf; jsadflkjasd fjksadf l;ksdajfl;ks dfjlks;df jsadlkfj asdfjks adfjksadflk sdajflks adfjlk;asdfjsdalk;f jsadlkf;jsad lfjsadfl ;sadjfl;ksad jflsdk;fj sdal;fj asdl;fjsadl;fkasjdf;l asd